src/components/App.tsx

```diff
@@ -10,11 +10,11 @@ interface Category {
 interface Endeavor {
 	id:number;
 	label:string;
-	category:number; // id from Category
+	category_id:number;
 }
 
 interface ActivityEndeavor {
-	id:number; // id from Endeavor
+	endeavor_id:number;
```

Comment:
I like this, but let's discuss the use of `endeavor_id` vs `endeavorId` or `endeavorID`. Wouldn't the the best practice for JavaScript to be to use camel case?